docs/source/notes/numpy_compatibility.rst

```diff
@@ -0,0 +1,221 @@
+.. _numpy_compatibility:
+
+NumPy Compatibility
+=====================
+
+.. note::
+
+   **TL;DR:**
+
+   - Be explicit about obtaining and using NumPy arrays, or
+   - Avoid operators (``-``, ``+``, ``*``, ``/``) in favor of methods
+     (``np.subtract``, ``np.add``, ``np.multiply``, ``np.divide``)
```

Comment:
The idea was that since `torch` operators don't work with ndarrays, if one would like to not worry about the asymmetry of `a+x` vs `x+a` where `a->np.ndarray` and `x->torch.tensor`, using the methods might be a viable option, but I think it'd be best to drop it.